frontend/src/concepts/pipelines/apiHooks/mlmd/useExecutionsFromContext.ts

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { Context, Execution, GetExecutionsByContextRequest } from '~/third_party/mlmd';
+import { FAST_POLL_INTERVAL } from '~/utilities/const';
+import useFetchState, {
+  FetchState,
+  FetchStateCallbackPromise,
+  NotReadyError,
+} from '~/utilities/useFetchState';
+
+export const useExecutionsFromContext = (
+  context: Context | null,
```

Comment:
Is it not confusing to have the concept of React's Context and also Context from MLMD? Maybe we should alias Context as `MlContext` or `MlmdContext` and name these functions `useExecutionsFromMlContext` or `useExecutionsFromMlmdContext`?

Reply:
And looking at your other file, it appears this was done to an extent already (e.g. `useMlmdContext`).